celesta-java/src/main/java/ru/curs/celesta/AppSettings.java

```diff
@@ -0,0 +1,22 @@
+package ru.curs.celesta;
+
+import java.util.*;
+
+public class AppSettings extends BaseAppSettings {
+
+    private final Set<String> celestaScan = new LinkedHashSet<>();
+
+    public AppSettings(Properties properties) {
+        super(properties);
+
+        String celestaScanProperty = properties.getProperty("celestaScan");
+        if (celestaScanProperty != null)
+            Arrays.stream(celestaScanProperty.split(","))
+                    .map(String::trim)
+                    .forEach(celestaScan::add);
+    }
+
+    public Set<String> getCelestaScan() {
+        return Collections.unmodifiableSet(celestaScan);
```

Comment:
Since we never modify CelestaScan set, it can be converted to unmodifiableSet once and for all on line 16, instead of creating wrapper object each time we request for this property ))